docs/apache-airflow/security/security_model.rst

```diff
@@ -61,15 +61,15 @@ model.
 DAG Authors
 ...........
 
-They can upload, modify, and delete DAG files. The
+They can create, modify, and delete DAG files. The
 code in DAG files is executed on workers and in the DAG File Processor. Note
 that in the simple deployment configuration, parsing DAGs is executed as
 a subprocess of the Scheduler process, but with Standalone DAG File Processor
 deployment managers might separate parsing DAGs from the Scheduler process.
 Therefore, DAG authors can create and change code executed on workers
 and the DAG File Processor and potentially access the credentials that the DAG
 code uses to access external systems. DAG Authors have full access
-to the metadata database and internal audit logs.
```

Comment:
I'm hesitant to start enumerating everything you can do with db access - audit logs is just 1 thing.

Reply:
It was there mostly with the reasoning that audit logs are somewhat important and can be used to "cover tracks" - but well, we do not have "true" unmodifiable, write-only audit logs anyway, so that is not needed here anyway.